src/pint/models/timing_model.py

```diff
@@ -3024,13 +3025,25 @@ def alias_to_pint_param(self, alias):
                 first_init_par = None
                 # Handle the case of start index from 0 and 1
                 for start_idx in [0, 1]:
-                    first_init_par_alias = prefix + "{1:0{0}}".format(fmt, start_idx)
+                    # Ensure compatibility with tempo-style "DM001"
+                    if "DM0" in alias:
+                        first_init_par_alias = prefix + "{1:0{0}}".format(
```

Comment:
Might be good to change the string formatting here to use f-strings.  They are a lot easier to read.  Same with the string formatting in the "else" case below.

Reply:
Just so you know, I pushed a change just now that only applies the old black version, it doesn't address anything else. I'll unmark as WIP when I've made all changes and added tests.